iree/test/e2e/xla_ops/rng.mlir

```diff
@@ -0,0 +1,34 @@
+func @rng_1d() {
```

Comment:
Let's rename this file to `rng_uniform` which matches the testing operation name?

Reply:
done